src/test/java/com/puppycrawl/tools/checkstyle/checks/javadoc/WriteTagCheckTest.java

```diff
@@ -58,32 +58,33 @@ public void testDefaultSettings() throws Exception {
     @Test
     public void testTag() throws Exception {
         final String[] expected = {
-            "15: " + getCheckMessage(MSG_WRITE_TAG, "@author", "Daniel Grenner"),
+            "21: " + getCheckMessage(MSG_WRITE_TAG, "@author", "// violation 3 lines" +
+                    " below 'Must contain non-whitespace characters.'"),
```

Comment:
you do not need to change expected.
look at https://github.com/checkstyle/checkstyle/commit/9857df8546c56f48b68fc5e930caffcc70eafe01 and other example of same type of update to see what others are doing.